### PR TITLE
[12.0][MIG] partner_communication_press: migration to 12.0

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

File: partner_communication_press/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Communication Press',
+
+    'summary': """
+    Add information in your contacts to know if you need to send
+    advertising to your contacts and which type of advertising.
+    """,
+    'description': """
```
Comment (vvrossem) [inline partner_communication_press/__manifest__.py]: remove `description`

File: partner_communication_press/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    """,
+
+    'author': 'Coop IT Easy, Rémy Taymans',
```
Comment (vvrossem) [inline partner_communication_press/__manifest__.py]: `Coop IT Easy SCRLfs`

File: partner_communication_press/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    'license': 'AGPL-3',
+    'version': '12.0.1.0',
```
Comment (vvrossem) [inline partner_communication_press/__manifest__.py]: `12.0.1.0.0`

File: partner_communication_press/models/partner.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    communication_press = fields.Boolean("Press ?")
+    communication_press_type_ids = fields.Many2many(
+        'res.partner.communication.press.type',
+        string="Press Type"
+    )
+
+    @api.multi
+    def write(self, vals):
+        for rec in self:
+            if 'communication_press' in vals:
```
Comment (vvrossem) [inline partner_communication_press/models/partner.py]: `if 'communication_press' in vals and not vals['communication_press']:`

**Review by vvrossem — CHANGES_REQUESTED** [review]
add yourself as Contributor :)

File: partner_communication_press/models/partner.py (hunk 4 above)
Comment (huguesdk) [reply partner_communication_press/models/partner.py]: done. funny thing is that with black formatting, this results in 2 extra lines compared to before the change. ☺

**Comment by huguesdk** [comment]
> add yourself as Contributor :)

since rémy was removed from the authors list, i added him to the contributors, putting him first. is that ok, or do you usually do it differently?

**Review by vvrossem — APPROVED** [review]

**Pull request merged**